lib/devices/ios/simulator.js

```diff
@@ -481,15 +481,31 @@ Simulator.prototype.getBuiltInAppDir = function (cb) {
     var appDir = path.resolve(xcodeDir, "Platforms/iPhoneSimulator.platform/" +
                               "Developer/SDKs/iPhoneSimulator" +
                               this.platformVer + ".sdk/Applications/");
-    fs.stat(appDir, function (err, s) {
+    var testDir = function (dir, ncb) {
+      fs.stat(appDir, function (err, s) {
+        if (err) {
+          ncb(err);
+        } else if (!s.isDirectory()) {
+          ncb(new Error("Could not load built in applications directory"));
+        } else {
+          logger.debug('Found app dir: ' + appDir);
+          ncb(null, dir);
+        }
+      });
+    };
+    testDir(appDir, function (err, dir) {
       if (err) {
-        cb(err);
-      } else if (!s.isDirectory()) {
-        cb(new Error("Could not load built in applications directory"));
+        // try out other location
+        logger.debug('Unable to find app dir ' + appDir);
+        appDir = path.resolve("/Library/Developer/CoreSimulator/" +
```

Comment:
Yes. The first one needs the escape because of the subsequent `chown` that will fail without it. The second is handled fine within Appium itself.